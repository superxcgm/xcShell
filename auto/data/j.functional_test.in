mkdir foo
j foo
j ..
j fo
j ..
j /
j /tmp/xcShell_functional_test/foo
j ..
j /
j fo
j ..