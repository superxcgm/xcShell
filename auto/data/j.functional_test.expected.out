xcShell_functional_test > mkdir foo
xcShell_functional_test > j foo
foo > j ..
xcShell_functional_test > j fo
foo > j ..
xcShell_functional_test > j /
/ > j /tmp/xcShell_functional_test/foo
foo > j ..
xcShell_functional_test > j /
/ > j fo
foo > j ..
xcShell_functional_test > 